starwars/src/App.js

```diff
@@ -1,40 +1,24 @@
 import React, { Component } from 'react';
 import './App.css';
-import CharCard from './Components/CharCard/CharCard';
-import PropTypes from 'prop-types';
+import PeopleList from './Components/PeoplesList/PeoplesList';
 
 class App extends Component {
   constructor() {
     super();
-    this.state = {
-      starwarsChars: []
-    };
-  }
-  componentDidMount() {
-    // feel free to research what this code is doing.
-    // At a high level we are calling an API to fetch some starwars data from the open web.
-    // We then take that data and resolve it our state.
-    fetch('https://swapi.co/api/people')
-      .then(res => {
-        return res.json();
-      })
-      .then(data => {
-        this.setState({ starwarsChars: data.results });
-      })
-      .catch(err => {
-        throw new Error(err);
-      });
```

Comment:
You need this api call here so that you can fetch the data that you are retrieving from the server.  If you go to https://swapi.co/api/people in the browser, you'll be able to see a preview of the api data that you can manipulate and display as required by the assignment. You've also removed all state from the component.  Without state, there is no way for data to be passed to rerender functional components.